test/tests/functional/pbs_multi_sched.py

```diff
@@ -699,3 +699,50 @@ def test_cmp_fairshare(self):
 
         user = self.scheds['sc3'].cmp_fairshare_entities(TEST_USER, TEST_USER2)
         self.assertEquals(user, str(TEST_USER))
+
+    def test_multisched_sched_cycle_length(self):
+        """
+        Test the scheduler attribute sched_cycle_length for multisched
+        """
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'},
+                            expect=True)
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'sched_cycle_length': '00:00:01'},
+                            id='default',
+                            expect=True)
+        a = {'resources_available.ncpus': 100}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        for _ in xrange(100):
+            j = Job(TEST_USER)
+            self.server.submit(j)
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'},
+                            expect=True)
+        t = int(time.time())
+        msg = ";Leaving the scheduling cycle:" +\
+              " Cycle duration of 1 seconds has exceeded" +\
+              " sched_cycle_length of 1 seconds"
+        self.scheduler.log_match(msg, max_attempts=10, starttime=t)
```

Comment:
By default max_attempts is 60 , no need to make it 10 or 15 . This may make test to fail on slower machines.

Reply:
@visheshh I believe default value is taken only when we don't use max_attempts. Since we are giving max_attempts=10 here I don't think default value apply here. Am I missing anything?

Reply:
@suresh-thelkar , Vishesh has a good point about slow machine. Any reason for adding max_attempts to 10 and not using default ,i.e, not defining any max_attempts?

Reply:
@suresh-thelkar : I was suggesting not use max_attempts

Reply:
max_attempts set to default.

Reply:
@anamikau, My Intentions is either we use max_attempts to longer value or depend on default value. Since @Bhagat-Rajput  now depends on default value I think it is fine.